pipeline/shared/duplicati.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+. "$( cd "$(dirname "$0")" ; pwd -P )/../docker-run/error_handling.sh"
+
+export DUPLICATI_ROOT="/application/"
+declare -a FORWARD_OPTS
+
+export UPDATE_SOURCE="${DUPLICATI_ROOT}/Updates/build/${BUILDTAG}_source"
+export UPDATE_TARGET="${DUPLICATI_ROOT}/Updates/build/${BUILDTAG}_target"
```

Comment:
I don't see `UPDATE_SOURCE` or `UPDATE_TARGET` referenced anywhere.  Is there a future use for these?

Reply:
Yes, I only added a part of the pipeline I constructed. Next stages are packaging, signing and deploying to docker. These are not relevant for PR's but will be for CD. I am awaiting feedback from Kenneth for this.

Reply:
:+1: